apps/architect-web/src/templates/care-support-networks.json

```
@@ -457,6 +457,11 @@
       "type": "Narrative",
       "label": "Your support map",
       "subject": { "entity": "node", "type": "supporter" },
+      "behaviours": {
```

Biome (2.5.0)

[error] The key behaviours was already declared.

(lint/suspicious/noDuplicateObjectKeys)